enhancements/windows-containers/windows-machine-config-operator-upgrades.md

```diff
@@ -113,9 +113,12 @@ associated with the node. The [Machine API Operator](https://github.com/openshif
 will drain and delete the node before the Machine deletion completes. It will
 then create a new Machine to reconcile the MachineSet replica count. WMCO will
 configure this new machine and apply the new version annotation. The max amount
-of unavailable nodes at a time will be dependent on the maxUnavailable field
-in the [MachineSet](https://github.com/openshift/machine-api-operator/blob/master/config/machineset.yaml)
-created for Windows machines. WMCO will not render more than the specified
+of unavailable nodes at a time will be dependent on the maxUnhealthy field defined
+internally by the WMCO. This field ensures that we only have maxUnhealthy number of
+nodes in Not Ready state during upgrades. Having limited number of unavailable nodes
```

Comment:
Is this specifically the kubelet `NotReady` status with no space? If so can you correct that?

Reply:
A Windows node is in Not Ready state if it is not configured completely i.e it is missing the version annotation set by WMCO. Updated the description to reflect this.

Reply:
We shouldn't be using `Not Ready` but rather `not ready` then.

Reply:
I have updated this to use `not ready`